pkg/storage/unified/search/bleve.go

```diff
@@ -1144,20 +1146,19 @@ func (b *bleveIndex) toBleveSearchRequest(ctx context.Context, req *resourcepb.R
 
 	// Add the sort fields
 	sorting := getSortFields(req)
-	searchrequest.SortBy(sorting)
-
-	// When no sort fields are provided, sort by score if there is a query, otherwise sort by title
-	if len(sorting) == 0 {
+	if len(sorting) > 0 {
+		searchrequest.SortBy(sorting)
+	} else {
+		// When no sort fields are provided, sort by score first if there is a query, otherwise sort only by title
 		if req.Query != "" && req.Query != "*" {
 			searchrequest.Sort = append(searchrequest.Sort, &search.SortScore{
 				Desc: true,
 			})
-		} else {
-			searchrequest.Sort = append(searchrequest.Sort, &search.SortField{
-				Field: resource.SEARCH_FIELD_TITLE_PHRASE,
-				Desc:  false,
-			})
 		}
+		searchrequest.Sort = append(searchrequest.Sort, &search.SortField{
```

Comment:
This change snuck in, I didn't mean to include it in this PR. Will remove it. (I think it's ok change, but doesn't belong here)